### PR TITLE
fix live mode API

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@testcafe-build-bot retest

File: src/live/controller.js [hunk 1]
```diff
@@ -20,10 +20,10 @@ class LiveModeController extends EventEmitter {
         this.logger         = new Logger();
         this.runner         = runner;
         this.lockKeyPress   = false;
+        this.fileWatcher    = null;
```
Comment (miherlosev) [inline src/live/controller.js]: Add `this.rl = null` to this constructor.

File: src/live/test-runner.js [hunk 2]
```diff
@@ -89,12 +89,23 @@ class LiveModeRunner extends Runner {
             .then(() => this.runTests(true));
 
 
-        return this._waitInfinite().then(() => {
-            this.preventRunCall = false;
-        });
+        return this._waitInfinite()
```
Comment (miherlosev) [inline src/live/test-runner.js]: `_waitInfinite` is bad name. When the promise of this method should be resolved? After infinite?

I propose to use `_waitUntilExit` name or something else.

Comment (miherlosev) [reply src/live/test-runner.js]: Yes. Let's do this later.

**Pull request merged**